Sensors/BNO/Bno.cpp

```diff
@@ -20,8 +24,12 @@ void BNO055::InitializeBNO() {
 }
 
 void BNO055::getBNOData() {
+    Uart uart;
+    uart.update();
     imu::Vector<3> euler = bno.getVector(Adafruit_BNO055::VECTOR_EULER);
```

Comment:
Make a single instance of the Uart object and intialize it in the constructor of intialization method. This code right now creates a new instance of `Uart` each time BNO Data is required.

Reply:
Already fixed this, it will be reflected on my new PR, thanks